shardingsphere-proxy/shardingsphere-proxy-bootstrap/pom.xml

```diff
@@ -88,6 +88,13 @@
             <artifactId>shardingsphere-tracing-opentracing</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.10</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Please do not add dependency for test only.
It is better to discuss of adding new dependency before do it.

Reply:
removed